calculator_data.yml

```diff
@@ -0,0 +1,70 @@
+---  # document start
+
+calendar_id : vakt@jodiskmuseumoslo.no
+
+email_sender : mejlzatestiranje69@gmail.com
+
+email_recipients :
+  - xlr80000@gmail.com
+  - markopetrovic@outlook.com
+
+default_date_day: 10
+
+bank-holidays:
+  - Sun
+  - Sat
+  - May 17
+  - Dec 24
+  - Dec 25
+  - Jan 01
+  - Apr 25 # test
+  - Apr 18, 2019
+  - Apr 19, 2019
+  - May 30, 2019
+  - Apr 09, 2020
+
+wage_rules:
+  - rule:
+     dow: holidays
+     formula: hr + 55
+  - rule:
+     dow: weekdays
+     start: 8
+     end: 17
+     formula: hr
+  - rule:
+     dow: weekdays
+     start: 17
+     end: 20
+     formula: hr + 15
+  - rule:
+     dow: weekdays
+     start: 20
+     end: 8
+     formula: (1 + 45./100.)*hr
+
+employees :
```

Comment:
These people are real people, should be kept out of this repo.

It is ok to commit a file like this, but please use some bogus names.

The real (production) config should be kept out of the repo.

Please read my comment about unit tests.